swift/src/Ice/ObjectAdapterI.swift

```diff
@@ -24,6 +22,7 @@ final class ObjectAdapterI: LocalObject<ICEObjectAdapter>, ObjectAdapter, ICEDis
     }
 
     private let communicator: Communicator
+    private let servantManager: ServantManager
```

Comment:
With the addition of setAdminId below, we no longer need to expose ServantManager to any other class - it's cleaner but doesn't change much otherwise.